flight/core/Dispatcher.php

```diff
@@ -71,7 +71,6 @@ protected function runPreFilters(string $eventName, array &$params): self
 
     /**
      * @param array<int, mixed> &$params
-     * @param mixed &$output
```

Comment:
When I merged your previous PR, this line was being thrown in phpstan

Reply:
![Screenshot_20240209-115037](https://github.com/flightphp/core/assets/109766973/65746b98-59dc-4349-ac66-e4c693ae609a)